src/content/quickstart/openshift/_index.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2020-02-21T13:36:18+01:00
+title: "OpenShift (AWS)"
+---
+
+## AWS
+
+### openshift-install and pull-secret
+
+Download the **openshift-install** and _oc_ tools, and copy your _pull secret_ from:
+
+> https://cloud.redhat.com/openshift/install/aws/installer-provisioned
+
+
+### Make sure you're logged in aws cli
+
+```bash
+$ aws configure
+AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
```

Comment:
No need to show specific id and key in Readme.

Reply:
I will change it for "...." this is copy&paste from the AWS guide to do this. They are just example IDs providen by amazon.